components/script/dom/clipboard.rs

```diff
@@ -95,6 +172,71 @@ impl ClipboardMethods<crate::DomTypeHolder> for Clipboard {
     }
 }
 
+impl RoutedPromiseListener<Result<String, String>> for Clipboard {
+    fn handle_response(
+        &self,
+        response: Result<String, String>,
+        promise: &Rc<Promise>,
+        _can_gc: CanGc,
+    ) {
+        let trusted_promise = TrustedPromise::new(promise.clone());
+        let text = response.unwrap_or_default();
+
+        // Step 3.4 Queue a global task on the clipboard task source,
+        // given realm’s global object, to perform the below steps:
```

Comment:
So actually we are [already in a task](https://github.com/servo/servo/blob/539ca272843bd0937bc7ed48d6103a6338d2a3c9/components/script/routed_promise.rs#L72) here, but using the wrong task source. So I suggest you remove the task queuing and just leave a TODO to use the right task source depending on https://github.com/servo/servo/issues/36825